linux/net/rina/rmt.c

```diff
@@ -881,35 +915,52 @@ static void send_worker(unsigned long o)
                         continue;
                 }
 
-                /* there are more than 1 pdu in the queue, we will hace to
-                 * reschedule */
-                if (atomic_read(&n1_port->n_sdus) >= MAX_PDUS_SENT_PER_CYCLE)
-                        reschedule++;
-
                 rcu_read_lock();
                 if (ps && ps->rmt_next_scheduled_policy_tx) {
+                	pdus_sent = 0;
+                	ret = 0;
                         do {
-                                pdu = ps->rmt_next_scheduled_policy_tx(ps, n1_port);
-                                if (pdu) {
+                        	pdu = ps->rmt_next_scheduled_policy_tx(ps, n1_port);
```

Comment:
This line is > 80  and several others as well in this function (line 929, 931, 935, 936, 949). Please fix it.